packages/components/src/utils/property-checkers/index.ts

```diff
@@ -1,15 +1,20 @@
-import { emptyOr } from './empty-or';
+import { requiredAnd } from './required-and';
 import { checkOneOf } from './check-one-of';
 import { checkPattern } from './check-pattern';
 import { checkType } from './check-type';
 import { checkUrl } from './check-url';
+import { emptyOr } from './empty-or';
 
 export const checkEmptyOrOneOf = emptyOr(checkOneOf);
 export const checkEmptyOrPattern = emptyOr(checkPattern);
 export const checkEmptyOrType = emptyOr(checkType);
 export const checkEmptyOrUrl = emptyOr(checkUrl);
 
-export * from './check-non-empty';
+export const checkRequiredAndOneOf = requiredAnd(checkOneOf);
+export const checkRequiredAndPattern = requiredAnd(checkPattern);
+export const checkRequiredAndType = requiredAnd(checkType);
+export const checkRequiredAndUrl = requiredAnd(checkUrl);
+
 export * from './check-one-of';
 export * from './check-pattern';
 export * from './check-type';
```

Comment:
Now that we have the `checkEmptyOr*` and `checkRequiredAnd` functions, basic functions should no longer be used in components therefore their export can be removed.
```suggestion
```